IDEAS/Buildings/Data/Constructions/UninsulatedFloor.mo

```diff
@@ -0,0 +1,17 @@
+within IDEAS.Buildings.Data.Constructions;
+record UninsulatedFloor "Construction data of Uninsulated Floor"
+  extends IDEAS.Buildings.Data.Interfaces.Construction(incLastLay=IDEAS.Types.Tilt.Floor,
```

Comment:
Start `incLastLay` at a new line to improve readibility and I would also add a possible `locGain`.